adblock-extras.css

```diff
@@ -41,10 +41,15 @@ div[class^=guildIconContainer_]:has(svg[class^=boostedGuildIconGem_]) {
 }
 
 /* Quests icon */
-svg[class^=questsIcon_] {
+svg[class^=questsIcon_] {
   display: none;
 }
 
+/* "Quest Completed" badge in profiles */
+div[aria-label="Completed a Quest"] {
```

Comment:
here's a sample of a container of badges:

```html
<div class="container_b64375" aria-label="User Badges" role="group">
   <div aria-label="Active Developer"><a class="anchor_af404b anchorUnderlineOnHover_af404b" href="https://support-dev.discord.com/hc/en-us/articles/10113997751447?ref=badge" rel="noreferrer noopener" target="_blank" role="button" tabindex="0"><img alt=" " aria-hidden="true" src="https://cdn.discordapp.com/badge-icons/6bdc42827a38498929a4920da12695d9.png" class="badge_b64375"></a></div>
   <div aria-label="Subscriber since Dec 3, 2021"><a class="anchor_af404b anchorUnderlineOnHover_af404b" href="https://discord.com/settings/premium" rel="noreferrer noopener" target="_blank" role="button" tabindex="0"><img alt=" " aria-hidden="true" src="https://cdn.discordapp.com/badge-icons/2ba85e8026a8614b640c2837bcdfe21b.png" class="badge_b64375"></a></div>
   <div aria-label="Server boosting since Dec 9, 2021"><a class="anchor_af404b anchorUnderlineOnHover_af404b" href="https://discord.com/settings/premium" rel="noreferrer noopener" target="_blank" role="button" tabindex="0"><img alt=" " aria-hidden="true" src="https://cdn.discordapp.com/badge-icons/ec92202290b48d0879b7413d2dde3bab.png" class="badge_b64375"></a></div>
   <div aria-label="Originally known as Jimbo#9153"><a class="anchor_af404b anchorUnderlineOnHover_af404b" role="button" tabindex="0"><img alt=" " aria-hidden="true" src="https://cdn.discordapp.com/badge-icons/6de6d34650760ba5551a79732e98ed60.png" class="badge_b64375"></a></div>
   <div aria-label="Completed a Quest"><a class="anchor_af404b anchorUnderlineOnHover_af404b" href="https://discord.com/discovery/quests" rel="noreferrer noopener" target="_blank" role="button" tabindex="0"><img alt=" " aria-hidden="true" src="https://cdn.discordapp.com/badge-icons/7d9ae358c8c5e118768335dbe68b4fb8.png" class="badge_b64375"></a></div>
</div>
```

hopefully this works for your client:

```suggestion
div:has(> a[href="https://discord.com/discovery/quests"]) {
```

i'm not a css pro, but i believe `:has(> foo)` selects an element that has an _immediate_ child `foo`.